types/cordova/index.d.ts

```diff
@@ -80,3 +80,13 @@ declare var cordova: Cordova;
 declare module 'cordova' {
     export = cordova;
 }
+
+interface Navigator {
+    /** This plugin displays and hides a splash screen during application launch. */
+    splashscreen: {
+        /** Dismiss the splash screen. */
+        hide(): void;
+        /** Displays the splash screen. */
+        show(): void;
+    }
+}
```

Comment:
is this API optional? that is, only when 'splashscreen' plugin is used? (sorry, I"m don't know Cordowa details)

Reply:
It used to be optional, it's since moved into the base Cordova package, this is why I am submitting the PR. There are more links in my original comment.

[This is supported on all of the Platforms.](https://cordova.apache.org/docs/en/latest/reference/cordova-plugin-splashscreen/#supported-platforms)

Reply:
sorry for missing linked docs